02.calendar/calender.rb

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env ruby
+#
+# ABOUT
+# ・コマンドラインで実行できるカレンダー表示スクリプトです
+# Usage
+# ・月と年を以下オプションの引数で設定し、該当年月のカレンダーを表示させる
+# -m : 1,2,3,4,5,6,7,8,9,10,11,12
+# -y : yyyy
+
+require 'date'
+require 'optparse'
+
+# デフォルトの表示年月を、今月に設定
+print_date = Date.today
+argument_month = print_date.month
+argument_year = print_date.year
+
+# コマンドライン引数の処理
+opt = OptionParser.new
+params = {}
+opt.on('-m [VALUE]') { |v| params['m'] = v }
+opt.on('-y [VALUE]') { |v| params['y'] = v }
+opt.parse!(ARGV)
+
+# paramsに値が入っている場合、コマンドライン引数をintegerに変換しつつ、変数に上書き
+if params.empty? == false
```

Comment:
rubyの式は全てtrueかfalseとして評価されます。
ですので、ifの条件式でtrueやfalseと比較することは基本的に無いと考えて良いと思います。

```suggestion
unless params.empty?
```